### ISSUE TITLE
Sprite Icon release issue

### ISSUE THREAD

**Issue by haritop** [jmoenig/Snap--Build-Your-Own-Blocks#2164]
When you select "release" from the sprite icon menu, the sprite icon disappears and the clone is not released from his parent.

A second issue, is that when you create a clone via "parent" in the sprite icon menu, then the inhereted methods do not appear in the clones palette, most of the times.

**Comment by brianharvey** [comment]
I don't know about the second one, but the first is the intended behavior.  It's just that Jens picked a horrible name for it.  What that menu item means is "make temporary":  The sprite does not appear in the sprite corral and is deleted when the stop sign icon is clicked.  It's the opposite of "edit," which really means "make permanent."

Did you close this on purpose?

**Comment by haritop** [comment]
Yes i did, when I looked at the code myself and saw the comment in the function.
I agree it is a little confusing. I would expect "release" to work as setting the parent to none.
As for the second, i think that the palettes of the clone are not instantly refreshed when setting the parent to some sprite. 
Anyway, thank you for your respond professor Harvey.
